openedx/core/djangoapps/user_api/api.py

```diff
@@ -162,6 +163,15 @@ def _apply_third_party_auth_overrides(request, form_desc):
                     }
                 )
 
+            if current_provider.other_settings:
+                other_settings = json.loads(current_provider.other_settings)
+                if 'PROVIDER_READ_ONLY_FIELDS' in other_settings:
+                    for field in other_settings['PROVIDER_READ_ONLY_FIELDS']:
+                        form_desc.override_field_properties(
+                            field,
+                            restrictions={"readonly": "readonly"}
+                        )
```

Comment:
We don't need this logic added to the login form, because email/fullname etc. are not modifiable from there.  And since [this version of `_apply_third_party_auth_overrides` is only called by `get_login_session_form`](https://github.com/open-craft/edx-platform/blob/5f6d3e3dd0715c29a07f1927c1905ea46075c17f/openedx/core/djangoapps/user_api/api.py#L133), this change can be removed.

Reply:
Done